app/assets/stylesheets/application/strategy.scss

```diff
@@ -3,7 +3,201 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
 @media screen and (max-width: $medium) {
-	#strategy_comment {
-		margin: 0;
-	}
-}
\ No newline at end of file
+  #strategy_comment {
+    margin: 0;
+  }
+}
+
+@media only screen and (min-width: $medium){
+  #myBtn{
+    border: 1px solid $secondary-text-color;
+    box-shadow: 0 0 2px $secondary-text-color inset;
+    color: $secondary-text-color;
+    background: #fff;
+    font: 20px "Lato";
+    margin-right: 20px;
+    width: 217px;
+    padding: 9px;
+    margin-top: -120px;
+  }
+  .confirmbtn{
+    border: 1px solid $secondary-text-color;
+    box-shadow: 0 0 2px $secondary-text-color inset;
+    color: $secondary-text-color;
+    background: #fff;
+    font: 20px "Lato";
+    width: 217px;
+    padding: 9px;
+    display: block;
+    margin: 30px 0px;
+  }
+  .confirmbtn a{
+    color: $secondary-text-color !important;
+    border-bottom: none !important;
+  }
+  .modal {
```

Comment:
This will also solve the issue of the modal not being responsive e.g.
![image](https://user-images.githubusercontent.com/3010728/28237315-485818cc-68f1-11e7-91f1-00697b20859d.png)